src/sentry/monitors/models.py

```diff
@@ -230,6 +235,18 @@ def get_status_display(self) -> str:
         return "active"
 
 
+@receiver(pre_save, sender=Monitor)
+def check_organization_monitor_limits(sender, instance, **kwargs):
+    if (
+        instance.pk is None
+        and sender.objects.filter(organization_id=instance.organization_id).count()
+        > MONITOR_ORG_LIMIT
+    ):
+        raise Exception(
+            f"Organization has created the maximum number of Monitors: {MONITOR_ORG_LIMIT}"
```

Comment:
I wonder if we have some limit related exceptions we could raise and catch in the API?

cc @wedamija

Reply:
I looked for that but didn't find anything convincing...also this is on the model object so I wasn't sure how exactly it would get returned to through the API like what handlers we have

Reply:
Yeah I do something like this elsewhere but we don't raise a specific exception or anything